src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/TaskTransformer.kt

```diff
@@ -90,4 +91,12 @@ class TaskTransformer(
     entity.booking !== null -> TaskStatus.complete
     else -> TaskStatus.notStarted
   }
+
+  private fun transformUserOrNull(userEntity: UserEntity?): ApprovedPremisesUser? {
+    return if (userEntity == null) {
+      null
+    } else {
+      userTransformer.transformJpaToApi(userEntity, ServiceName.approvedPremises) as ApprovedPremisesUser
+    }
+  }
```

Comment:
This could be expressed more idiomatically as:

```suggestion
  private fun transformUserOrNull(userEntity: UserEntity?) = userEntity?.let {
    userTransformer.transformJpaToApi(it, ServiceName.approvedPremises) as ApprovedPremisesUser
  }
```